rpcs3/Emu/RSX/GL/GLTexture.cpp

```diff
@@ -319,6 +319,14 @@ namespace rsx
 				std::vector<gsl::byte> staging_buffer)
 			{
 				int mip_level = 0;
+				if (is_compressed_format(format))
+				{
+					//Compressed formats have a 4-byte alignment
+					//TODO: Verify that samplers are not affected by the padding
+					width = align(width, 4);
+					height = align(height, 4);
```

Comment:
This makes OpenGL draw the texture in Wolfenstein, but it is wrong (there should be an alpha gradient left and right):
![grafik](https://cloud.githubusercontent.com/assets/1445782/24424766/0fa42db6-1402-11e7-89e3-bc6f65d7e7a5.png)

On Vulkan the rendering of such a texture was actually correct before, now it crashes in the vk::image constructor.